tests/jobs/test_scheduler_job.py

```diff
@@ -2034,7 +2034,7 @@ def test_dagrun_root_fail_unfinished(self):
             ti = dr.get_task_instance('test_dagrun_unfinished', session=session)
             ti.state = State.NONE
             session.commit()
-        dr_state = dr.update_state()
+        dr_state = dr.state
```

Comment:
This looks suspect -- I would expect it to be
```suggestion
        dr.update_state()
        dr_state = dr.state
```